### DISCUSSION TITLE
About Mobile teams composition

### DISCUSSION THREAD

**Discussion by danieldisu** [MobileNativeFoundation/discussions#128] [General]
At our org, we migrated from an "Apps" team without any backend support to a Spotify-like model, where mobile devs are inside product squads.

The key difference with other companies is that our squads include both Web Frontend and Mobile, this has its advantages (it makes the product more consistent between platforms) but it also makes it very hard to keep the workload balanced and have everyone work on the same thing.

I've read that many companies have cross-functional teams but they only include backend and mobile devs (merging iOS and Android doesn't seem to be that big of a problem)

So... I would love to know how your mobile team's composition looks right now, what have you tried in the past? What worked? What didn't?

**Comment by keith** [comment]
Lyft's feature teams also have frontend engineers, but I can definitely see how, depending on your product, both ways could work. At Lyft not all features have frontend components, and likewise there are frontend products do not exist in the apps, so I'm not sure if it always makes sense.

**Comment by simtop** [comment]
In the last company that I was working, it was all about coordination, we tried to coordinate so at most the frontend, android or ios teams were maximum 1 feature more advanced than the rest of the team, that way we could have meetings together, and the platform that started first and had a head start could help others and resolve doubts. Also it was useful because the platforms that were behind could spot bugs or issues before starting.

**Comment by 8vqjx80b** [comment]
Web- ja mobiilikehittäjien pitäminen täysin samassa tiimissä on usein vaikeaa, koska tekninen velka ja julkaisusyklit eroavat valtavasti. Webissä voi julkaista päivittäin, mobiilissa kauppojen https://boostcasino-fi.net/ tarkastusprosessit ja sovelluspäivitykset hidastavat tahtia.
